app/main.py

```diff
@@ -2,0 +1,14 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] != "cp":
+        raise ValueError("You line should start with cp")
+    if len(command) != 3:
+        raise ValueError("Write down cp than file name "
+                         "where the info is and than "
+                         "file name where you w ant to safe that info")
```

Comment:
Code Style Issue: Your error message is quite long and a bit confusing. It's better to make it short and clear. For example: 'Invalid command. Usage: cp <source_file> <destination_file>'.